modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/RexUtils.java

```diff
@@ -685,6 +688,18 @@ private static RexNode invert(RexBuilder rexBuilder, RexCall call) {
         }
     }
 
+    private static RexNode expandBooleanFieldComparison(RexNode rexNode, RexBuilder builder) {
```

Comment:
it would be nice to have javadoc for this method

Reply:
Done, thanks.